### ISSUE TITLE
Tarlov Cyst

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4902]
**Preferred term label:**
Tarlov Cyst

**Synonyms**

**Definition (free text, please give PubMed ID)**
see https://www.aans.org/en/Patients/Neurosurgical-Conditions-and-Treatments/Tarlov-Cyst

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

add annotation: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3616632/

**Your nano-attribution (ORCID)**
